scripts/import-proto-and-deps.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -eu
+
+TMPDIR="$(mktemp -d)"
+git clone https://github.com/archway-network/archway --depth=1 --single-branch "$TMPDIR"
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+[ -d "$REPO_ROOT/proto" ] && rm -rf "$REPO_ROOT/proto"
+cp -r "$TMPDIR/proto" "$REPO_ROOT"
+cp -r "$TMPDIR/third_party/proto" "$REPO_ROOT"
```

Comment:
You don't need to duplicate the script. Let's add this single line to the existing one and delete this file.